script/clean.py

```diff
@@ -0,0 +1,73 @@
+import csv
+from os import sep
+import unidecode
+import pandas as pd
+
+
+path = ''
+file = open(path + "amajambo.csv")
+csvreader = csv.reader(file)
+
+#output files
+f1 = open('amajambo_cleanedV1.csv', 'w', newline='')
+
+f2 = open('amajambo_5letters.csv', 'w', newline='')
+
+
+rows = []
+for row in csvreader:
+    rows.append(row)
+
+flat_list = []
+for sublist in rows:
+    for item in sublist:
+        flat_list.append(item)
+
+words_list = [item.upper() for item in flat_list]
```

Comment:
At the end I realised it is easier to play with lowercase words can you please change this as well ?

```suggestion
words_list = [item.lower() for item in flat_list]
```